packages/server/src/cache/keys.ts

```diff
@@ -0,0 +1,3 @@
+export const CACHE_KEYS = {
+  SITE_SETTINGS: "siteSettings_cache",
```

Comment:
@TaranaKhanna Refer to `packages/server/src/ee/services/boards/invalidateCache.ts` file as to how we defined the key names in terms of pattern.